numba_typing/arguments_annotation_default_func_class.py

```diff
@@ -0,0 +1,24 @@
+from inspect import signature, getfullargspec
+from typing import get_type_hints
+
+
+def get_arguments_annotation_default_func(method):
```

Comment:
What do you think about renaming the function to `get_func_annotations` and the argument name to `func`?